src/release.js

```diff
@@ -119,6 +121,19 @@ module.exports = async function ({ github, context, inputs }) {
       inputs
     )
 
+    const shouldNotifyOnTheIssue = /true/i.test(inputs['notify-on-the-issue'])
```

Comment:
make sure the input name change reflects here as well of course

Reply:
This has also been updated to reflect the name change